lib/topological_inventory/openshift/operations/core/service_catalog_client.rb

```diff
@@ -11,17 +11,13 @@ module Core
         class ServiceCatalogClient
           include Logging
 
-          attr_accessor :default_endpoint, :authentication, :connection_manager, :sleep_poll, :identity
+          attr_accessor :connection_manager, :sleep_poll, :source_id, :identity
+          attr_writer   :authentication, :default_endpoint
```

Comment:
Do these even need to be writers at this point?

Reply:
Nope, removed